src/ui/components/UploadScreen/Privacy.tsx

```diff
@@ -46,10 +46,10 @@ function FavIcon({ url }: { url: string }) {
   return (
     <div className="relative">
       <div className="flex">
-        <MaterialIcon>public</MaterialIcon>
+        <MaterialIcon className="invisible">public</MaterialIcon>
```

Comment:
This is here as a backup while the actual favicon loads. Was there a reason for keeping this invisible instead of just removing it altogether?

Reply:
Yeah, I had a lot of trouble with these @jaril. When I made them larger they looked blurry. Let's pair up and figure out how to fix this!